docs/guide/integrations/nuxt3.md

```diff
@@ -265,11 +265,6 @@ Configure `nuxt.config.ts` like the below:
  // https://v3.nuxtjs.org/api/configuration/nuxt.config
  export default defineNuxtConfig({
 +  vite: {
-+    resolve: {
-+      alias: {
-+        'vue-i18n': 'vue-i18n/dist/vue-i18n.runtime.esm-bundler.js'
-+      }
-+    },
```

Comment:
@Atinux 
remove `vite.resolve.alias` no longer, so It's set in `@intlify/unplugin-vue-i18n`.
https://github.com/intlify/bundle-tools/releases/tag/unplugin-vue-i18n%400.3.0